hadoop-project/pom.xml

```diff
@@ -1150,26 +1165,6 @@
         <artifactId>woodstox-core</artifactId>
         <version>${woodstox.version}</version>
       </dependency>
-      <dependency>
-        <groupId>org.codehaus.jackson</groupId>
-        <artifactId>jackson-mapper-asl</artifactId>
-        <version>${jackson.version}</version>
-      </dependency>
-      <dependency>
-        <groupId>org.codehaus.jackson</groupId>
-        <artifactId>jackson-core-asl</artifactId>
-        <version>${jackson.version}</version>
-      </dependency>
-      <dependency>
-        <groupId>org.codehaus.jackson</groupId>
-        <artifactId>jackson-jaxrs</artifactId>
-        <version>${jackson.version}</version>
-      </dependency>
-      <dependency>
-        <groupId>org.codehaus.jackson</groupId>
-        <artifactId>jackson-xc</artifactId>
-        <version>${jackson.version}</version>
-      </dependency>
```

Comment:
Jackson 1.x still exists in the dependency, so we need to keep this to fix the version to 1.19.3. Remove these when upgrading Avro.

Reply:
Btw even if we remove this, we will still have 1.19.3 version coming from Avro:
```
+-org.apache.hadoop:hadoop-common:3.4.0-SNAPSHOT
  +-org.apache.avro:avro:1.7.7
    +-org.codehaus.jackson:jackson-mapper-asl:1.9.13
```
However, given that avro should be upgraded independent from whether jackson1 is still required by other Hadoop dependencies, I think it makes sense to not remove this block. Hence, reverted this change.